dfvfs/vfs/attribute.py

```diff
@@ -25,6 +25,8 @@ class StatAttribute(object):
   """Attribute that represents a POSIX stat.
 
   Attributes:
+    device_identifier (Tuple[int, int]): device identifier (if special file),
```

Comment:
nit: '(if special file)' => , if block or character device file.

since FIFO/pipe and socket are "special files" as well, but don't have a device number

Reply:
Done.